data/index.html

```diff
@@ -133,40 +133,52 @@ <h5 class="text-center">USGS Topo Maps</h5>
             </a>
         </div>
     </div>
-    <div class="grid__col section alternate">
-        <h2>State Geographic Information Database (SGID)</h2>
-        <p class="readable">Established by state law in 1991, Utah’s SGID ensures that GIS map layers are developed in a coordinated fashion and shared openly. The SGID provides a one-stop access point to hundreds of data layers that were developed, aggregated, or acquired by state government. To the credit of the GIS community across the state, the SGID's data resources are among the most mature and extensive in the nation. </p>
-        <p class="readable">Today, interoperable GIS server technology has diminished the importance of a completely centralized GIS data repository, and it is critical that the latest, best quality GIS data resources are readily discovered and accessed. AGRC strives to ensure that this is true for SGID data layers that are available as downloadable shapefiles or live streaming data services. </p>
-    </div>
     <div class="grid__col section">
-        <h2>The Open SGID</h2>
-        <p class="readable">The Open SGID is a cloud-hosted PostGIS database containing the datasets from the SGID which are also available from the <a href="http://opendata.gis.utah.gov/">AGRC Open Data</a>. It is a publicly-available, direct database connection that can be utilized on a multitude of GIS platforms to access Utah spatial data. A <a href="https://github.com/agrc/open-sgid">GitHub repository</a> is used as the primary source of information for the Open SGID, and to catalog and address issues with explanations and or troubleshooting techniques. Please read, understand, and abide by the <a href="{% link about/policy/open-sgid/index.md %}">terms of use</a> prior to connecting.</p>
-        <p class="section-read-more">
-            <a href="{% link sgid/open-sgid/index.md %}" class="btn">Get connected</a>
+        <h2>Welcome to the State Geographic Information Database (SGID)</h2>
+        <p class="readable">The <a href="{% link sgid/index.html %}">SGID</a> aims to be a centralized repository of GIS datasets created or curated by state agencies. SGID datasets live in many different places depending on their type, availability, and popularity:
```

Comment:
**thought:** The word "repository" harkens me back to the days of SGID as only an SDE database. Perhaps the word "source" is a little more flexible, given the distributed nature of the SGID as we know it today?

Reply:
I changed it to "collection" because I think that reflects the idea that there's multiple technologies coming from multiple agencies (ie, lots of sources). Does that work for you?